src/util/testUtil.ts

```diff
@@ -39,8 +40,33 @@ export const TEST_OPTIONS = {
   registry: getRegistry('Kusama', 'kusama', 9999),
 };
 
+export const DEMOCRACY_TEST_OPTIONS = {
+  metadataRpc: democracyMetadataRpc,
+  registry: getRegistry('Kusama', 'kusama', 9999),
+};
+
+export const CC1_TEST_BASE_TX_INFO = {
```

Comment:
Good idea to give explicit names to objects. Could you rename the old `TEST_OPTIONS` to `KUSAMA_TEST_OPTIONS`?